src/places/mutations.js

```diff
@@ -38,3 +40,28 @@ export function useSavePlaceMutation () {
     },
   ))
 }
+
+// TODO what about websocket place type updates?
```

Comment:
Could be a nice backend addition, although not expecting them to change a lot, so maybe OK. And could make the query re-load them here and there with the right options...